bin/busted

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env resty
 
+ngx.RESTY_CLI = true
+
```

Comment:
That would lead to code duplication and would be quite sparse. Also as previously discussed, a more complicated solution for the same result, wouldn't do that.